common/utils.js

```diff
@@ -453,6 +453,17 @@ const getItsEdgeContract = (chainConfig) => {
     return itsEdgeContract;
 };
 
+const getItsEdgeChains = (config, excludeChainName) => {
+    return Object.keys(config.chains).filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
+};
+
+const parseTrustedChains = (config, trustedChains, excludeChainName) => {
```

Comment:
bad pattern to exclude chain name in the helper that's meant to parse names. exclusion should be done by the caller (but as per above comment, exclusion doesn't even make sense)